### PR TITLE
Increase critical value threshold

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Resolves: #2986

## Description

Test of randomness for causes sporadic failures by being to strict.

## Testing

CI/CD

## Documentation

No.

## Installation

No.

### CODE REVIEW

File: libs/runtime/unit/platform_unit_test.cpp [hunk 1]
```diff
@@ -1094,7 +1094,7 @@ TEST_CASE("get_authentication_id", "[platform]")
 
 #define SEQUENCE_LENGTH 1024 * 128
 #define CHI_SQUARED_STATISTIC_THRESHOLD \
-    3.841 // Critical value for Chi-squared test with 2 degrees of freedom with significance level of 0.05.
+    9.210 // Critical value for Chi-squared test with 2 degrees of freedom with significance level of 0.01.
```
Comment (shpalani) [inline libs/runtime/unit/platform_unit_test.cpp:1097]: For my understanding, how does the change in level

Comment (shpalani) [inline libs/runtime/unit/platform_unit_test.cpp:1097]: For my understanding, how does the change in level help?
Was there a test failure/problem already encountered?

Comment (Alan-Jowett) [reply libs/runtime/unit/platform_unit_test.cpp:1097]: Yes, see #2986

**Review by saxena-anurag — APPROVED** [review]

File: libs/runtime/unit/platform_unit_test.cpp (hunk 1 above)
Comment (shpalani) [reply libs/runtime/unit/platform_unit_test.cpp:1097]: The issue doesn't provide the Root Cause Analysis.

**Review by dv-msft — APPROVED** [review]

**Pull request merged**